spec/system/admin_adds_new_case_spec.rb

```diff
@@ -1,22 +1,66 @@
-require "rails_helper"
+require 'rails_helper'
 
-RSpec.describe "admin adds a new case", type: :system do
-  xit "is successful" do
-    admin = create(:casa_admin)
-    case_number = "12345"
+RSpec.describe 'admin adds a new case', type: :system do
+  let(:admin) { create(:casa_admin) }
+  let(:case_number) { '12345' }
+
+  before do
     sign_in admin
     visit root_path
-    click_on "Cases"
-    expect(page).to have_selector(".case-list")
+    click_on 'Cases'
+
+    click_on 'New Case'
+  end
+
+  context 'when all fields are filled' do
+    it 'is successful' do
+      fill_in 'Case number', with: case_number 
+      select '3', from: 'casa_case_court_date_3i'
+      select 'March', from: 'casa_case_court_date_2i'
+      select '2020', from: 'casa_case_court_date_1i'
+  
+      check 'Transition aged youth'
+      has_checked_field? 'Transition aged youth'
+  
+      click_on 'Create CASA Case'
+
+      expect(page.body).to have_content(case_number)
+      expect(page).to have_content('CASA case was successfully created.')
+      expect(page).to have_content('Next Court Date: Tuesday, 3-MAR-2020')
+      expect(page).to have_content('Transition Aged Youth: Yes')
+    end
+  end
 
-    click_on "New Case"
-    fill_in "Case number", with: case_number
+  context 'when non-mandatory fields are not filled' do
+    it 'is successful' do
+      fill_in 'Case number', with: case_number
+      click_on 'Create CASA Case'
+
+      expect(page.body).to have_content(case_number)
+      expect(page).to have_content('CASA case was successfully created.')
+      expect(page).to have_content('Next Court Date:')
+      expect(page).to have_content('Transition Aged Youth: No')
+    end
+  end
+
+  context 'when the case number field is not filled' do
+    it 'is successful' do
```

Comment:
I would change this and L58 to `it 'does not create a new case' do`

Reply:
I thought I had changed those parts. Thanks for the comment, I just corrected it according to your suggestion :)